lib/helpers.py

```diff
@@ -658,4 +658,4 @@ def _from_parquet_corrected(self, file_name, s3_file_name, fs, columns):
             _CSVHelpers._to_parquet(df=df, file_name=file_name)
             fs.put(file_name, s3_file_name)
 
-        return df
+        return df, [test_users, control_users]
```

Comment:
No need to use such obscene syntax :)
`df, test_users, control_users` = Pythonish